aries-site/src/data/wcag/components.json

```diff
@@ -12,6 +12,24 @@
       { "rule": "2.4.9", "status": "conditional" },
       { "rule": "3.2.1", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
+      { "rule": "1.4.11", "status": "passed" },
+      { "rule": "1.4.13", "status": "passed" },
```

Comment:
Since the Menu is opened by click rather than hover or focus I feel like this rule might not apply.